### ISSUE TITLE
Trying to log with console.error generates invalid JSON (wrapped in consola?)

### ISSUE THREAD

**Issue by gkatsanos** [nuxt/nuxt.js#8741]
```js
  $axios.onError((error) => {
    console.log({
      date: new Date().toISOString(),
      level: "error",
      message: error.message,
      error: { stack: error.stack, message: error.message }
    })
    app.$sentry.withScope((scope) => {
      scope.setExtra('route', route)
      app.$sentry.captureException(error)
    })
  })
```

returns `[ERROR] { ... }` instead of simply `{ date: 1/02/2021 , level: "error", "message": "404 not found" }`

**Comment by pi0** [comment]
Hi @gkatsanos. I've filled bug report https://github.com/nuxt-contrib/consola/issues/109 for consola. Meanwhile you can use provided workarounds :)

**Comment by pi0** [comment]
Issue fixed in [consola@2.15.1](https://github.com/nuxt-contrib/consola/releases/tag/v2.15.1). For fix you can use `yarn upgrade nuxt` or better recreating lock file.
